test/functionalTest/cases/0000_content_related_headers/in_out_formats.test

```diff
@@ -86,12 +86,12 @@ echo '================================================================='
 echo
 
 echo "1.4: Unrecognized Content-Type, Accepts text/plain: 415, response in XML"
-orionCurl --url "$url" --payload "${payload}" --in "text/plain" --out "text/plain"
+orionCurl --url "$url" --payload "${payload}" --in "text/plain" --out "text/plain" --payloadCheck xml
```

Comment:
Fixes in orionCurl.
If out format is json or xml, orionCurl does the payloadCheck (beautifier).
In this case the out-format is text/plain, but orionCurl STILL did the payloadCheck.
The new outputs for text/plain, that are just a string or a Number fail, of course.
I had to modify the *payload check mechanism* in orionCurl.
Here, the output is actually XML, and I want to force the payload check mechanism.